src/openms/include/OpenMS/ANALYSIS/DECHARGING/MetaboliteFeatureDeconvolution.h

```diff
@@ -125,6 +126,10 @@ namespace OpenMS
     ///      (Na+) -> (H+Na+)
     void inferMoreEdges_(PairsType& edges, Map<Size, std::set<CmpInfo_> >& feature_adducts);
 
+    void candidateEdges(FeatureMapType& fm_out, const Adduct& default_adduct, PairsType& feature_relation, Map<Size, std::set<CmpInfo_> >& feature_adducts);
```

Comment:
can you use the common notation for private functions and use tailing underscore for these two fxn, e.g. `candidateEdges_`